ConwaysGameOfLife.Tests/SourceData.cs

```diff
@@ -0,0 +1,16 @@
+namespace ConwaysGameOfLife.Tests
+{
+    public static class SourceData
+    {
+        public static Universe UniverseInitializedAllDeadCells()
+        {
+            var initData = new CellState[][]
+            {
+                new []  {CellState.Dead,   CellState.Dead,    CellState.Dead },
+                new []  {CellState.Dead,   CellState.Dead,    CellState.Dead },
+                new []  {CellState.Dead,   CellState.Dead,    CellState.Dead }
+            };
+            return new Universe(initData);
+        }
+    }
+}
```

Comment:
```suggestion
}

```
add newline to end of file - this is just for you to see a review suggestion and how you can commit it directly to the branch from github's UI